test/support/pages/SettingsPage.js

```diff
@@ -126,8 +142,17 @@ define(function (require) {
       var self = this;
       var selector = 'li.kbn-settings-tab.active a small';
 
-      return self.remote.setFindTimeout(defaultTimeout)
-      .findByCssSelector(selector).getVisibleText()
+      var getText = function () {
```

Comment:
Did you add this? Was there a reason for it? It looks like something I've done in the past; I suspect you can revert all the changes in SettingsPage.js